packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -129,6 +179,12 @@ To enable grouped batching:
     }
 ```
 
+## Note about performance and latency
+
+In terms of performance and impact on latency, Fluid does not make any guarantees or provide benchmarks as the effects of compression/chunking/grouped batching are coupled with the nature of the payloads and the configured `FlushMode`. Therefore, customers must perform the required measurements and adjust the settings according to their scenarios.
+
+In general, compression offers a trade-off between higher compute costs, lower bandwidth consumption and lower storage requirements, while chunking slightly increases latency due to the overhead of splitting an op, sending the chunks and reconstructing them on each client. Grouped batching heavily decreases the number of ops observed by the server (therefore decreasing the odds of throttling) and slightly decreases the bandwidth requirements as it merges all the ops in a batch into a single op and also eliminates the op envelope overhead.
```

Comment:
> Grouped batching heavily decreases the number of ops observed by the server (therefore decreasing the odds of throttling)

Does this mean that Fluid server implementations throttle based on the number of _ops_ (not http requests / websocket messages) in some time range? So the effect of batching and grouped batching are different in that regard? Also, is that supposed to be standard in any Fluid server implementation?

Reply:
Good callout. This only handles any possible op count throttling from server implementations.

Reply:
I don't think this should be specified TBH